tests/copilot_helper/fixtures/make_addons/expected/web/addons/my-aurora-db.yml

```diff
@@ -0,0 +1,265 @@
+              }
+              
+              if "dbClusterIdentifier" in master_user.keys():
+                  user_secret_string["dbClusterIdentifier"] = master_user["dbClusterIdentifier"]
+              
+              if "dbInstanceIdentifier" in master_user.keys():
+                  user_secret_string["dbInstanceIdentifier"] = master_user["dbInstanceIdentifier"]
+
+              conn = psycopg2.connect(
+                  dbname=master_user["dbname"],
+                  user=master_user["username"],
+                  password=master_user["password"],
+                  host=master_user["host"],
+                  port=master_user["port"]
+              )
+
+              cursor = conn.cursor()
+
+              response = {"Status": "SUCCESS"}
+
+              try:
+                  match event["RequestType"]:
+                      case "Create":
+                          create_db_user(conn, cursor, username, user_password, user_permissions)
+
+                          response = {
+                              **response,
+                              "Data": create_or_update_user_secret(ssm, user_secret_name, user_secret_string, event)
+                          }
+                      case "Update":
+                          create_db_user(conn, cursor, username, user_password, user_permissions)
+
+                          response = {
+                              **response,
+                              "Data": create_or_update_user_secret(ssm, user_secret_name, user_secret_string, event)
+                          }
+                      case "Delete":
+                          drop_user(cursor, username)
+
+                          response = {
+                              **response,
+                              "Data": ssm.delete_parameter(Name=user_secret_name)
+                          }
+                      case _:
+                          response = {"Status": "FAILED",
+                                      "Data": {"Error": f"""Invalid requestType of '${event["RequestType"]}'"""}}
+              except Exception as e:
+                  response = {"Status": "FAILED", "Data": {"Error": str(e)}}
+
+              cursor.close()
+              conn.close()
+
+              print(json.dumps(response, default=str))
+              cfnresponse.send(event, context, response["Status"], response["Data"], event["LogicalResourceId"])
+
+  myAuroraDbApplicationUser:
+    Type: 'Custom::myAuroraDbApplicationUser'
+    DeletionPolicy: Delete
+    UpdateReplacePolicy: Retain
+    Properties:
+      ServiceToken: !GetAtt 'myAuroraDbLambdaFunction.Arn'
+      CopilotApplication: !Sub "${App}"
+      CopilotEnvironment: !Sub "${Env}"
+      MasterUserSecret: 
+        Fn::ImportValue:
+          !Sub "${App}-${Env}-myAuroraDb-AuroraSecretName"
+      SecretDescription: !Sub  application user secret for ${AWS::StackName}
+      SecretName: !Sub '/copilot/${App}/${Env}/secrets/MY_AURORA_DB_APPLICATION_USER'
```

Comment:
Should be service namespaced, if multiple services connect to the DB, thinking a worker service and a beat service, both will attempt to overwrite the same secret.